File: XUCore/additions/NSToolbarAdditions.m
// 
// NSToolbarAdditions.m
// 
// Created by Charlie Monroe
// 
// Copyright (c) 2010-13 Fuel Collective, LLC. All rights reserved.
// 


#import "NSToolbarAdditions.h"


@implementation NSToolbar (NSToolbarAdditions)

-(NSToolbarItem*)itemWithIdentifier:(NSString*)identifier{
	NSArray *items = [self items];
	for (NSToolbarItem *item in items){
		if ([[item itemIdentifier] isEqualToString:identifier]){
			return item;
		}
	}
	return nil;
}

@end

File: XUCore/additions/NSToolbarAdditions.swift
//
//  NSToolbarAdditions.swift
//  XUCore
//
//  Created by Charlie Monroe on 11/8/15.
//  Copyright © 2015 Charlie Monroe Software. All rights reserved.
//

import Foundation

public extension NSToolbar {
	
	/// Returns item with identifier.
	public func item(withIdentifier identifier: String) -> NSToolbarItem? {
		return self.items.first(where: { $0.itemIdentifier == identifier })
	}
	
}
